browser/resources/settings/email_aliases_page/stories/index.tsx

```diff
@@ -0,0 +1,185 @@
+  emailAliasesBubbleLimitReached: 'You have reached the limit of 5 free email aliases. Click "Manage" to re-use or delete an alias.',
+})
+
+export default {
+  title: 'Email Aliases',
+}
+
+const demoData = {
+  email: 'username@brave.com',
+  aliases: [
+    {
+      email: 'horse.radish.record57@bravealias.com',
+      note: 'Alias for all my newsletters',
+    },
+    {
+      email: 'start.plane.division.laser42@bravealias.com',
+      domains: ['x.com'],
+      note: 'Twitter account'
+    },
+    {
+      email: 'racoon.pencil.test14@bravealias.com',
+      note: 'Marketplace email for Facebook'
+    }
+  ]
+}
+
+class MockMappingService implements MappingService {
+  accountEmail_: string
+  aliases_ : Map<string, Alias>
+  accountState_ : AccountState
+  accountRequestId_ : number
+  constructor(accountState: AccountState, accountEmail: string) {
+    this.accountState_ = accountState
+    this.accountEmail_ = accountEmail
+    this.aliases_ = new Map<string, Alias>();
+    for (const alias of demoData.aliases) {
+      this.aliases_.set(alias.email, alias)
+    }
+  }
+
+  async createAlias (email: string, note: string): Promise<void> {
+    const alias = { email, note }
+    this.aliases_.set(email, alias)
+  }
+
+  async getAliases (): Promise<Alias[]> {
+    return [...this.aliases_.values()]
+  }
+
+  async updateAlias (email: string, note: string, status: boolean): Promise<void> {
+    const alias = { email, note }
+    this.aliases_.set(email, alias)
+  }
+
+  async deleteAlias (email: string): Promise<void> {
+    this.aliases_.delete(email)
+  }
+
+  async generateAlias (): Promise<string> {
+    let generated: string = ''
+    do {
+      generated = "mock-" + Math.random().toString().slice(2,6) + "@bravealias.com"
+    } while (this.aliases_.has(generated))
+    await new Promise(resolve => setTimeout(resolve, 1000))
+    return generated
+  }
+
+  async getAccountEmail (): Promise<string | undefined> {
+    return this.accountEmail_
+  }
+
+  async requestAccount (accountEmail: string): Promise<void> {
+    this.accountState_ = 'AwaitingAccount'
+    this.accountRequestId_ = window.setTimeout(() => {
+      this.accountEmail_ = accountEmail
+      this.accountState_ = 'AccountReady'
+    }, 5000);
+  }
+
+  async onAccountReady (): Promise<boolean> {
+    while (this.accountState_ === 'AwaitingAccount') {
+      await new Promise(resolve => setTimeout(resolve, 250));
+    }
+    return this.accountState_ === 'AccountReady'
+  }
+
+  async cancelAccountRequest (): Promise<void> {
+    this.accountState_ = 'NoAccount'
+    window.clearTimeout(this.accountRequestId_)
+  }
+
+  async logout (): Promise<void> {
+    this.accountState_ = 'NoAccount'
+  }
+
+  async closeBubble (): Promise<void> {
+    console.log("closeBubble")
+  }
+
+  async fillField (fieldValue: string): Promise<void> {
+    console.log("fillField", fieldValue)
+  }
+
+  async showSettingsPage (): Promise<void> {
```

Comment:
return types are implied here, so no need to annotate these funcitons with them

Reply:
removed